css/styles.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+    font-family: 'Noto Sans';
+    src: url(../fonts/NotoSans-Regular.ttf);
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Noto Sans';
+    src: url(../fonts/NotoSans-Bold.ttf);
+    font-weight: bold;
+}
+
+body {
+    color: #424242;
+    font-family: 'Noto Sans', sans-serif;
+    font-size: 14px;
+    letter-spacing: .01em;
+    line-height: 1.3;
+    padding: 24px;
+    max-width: 800px;
```

Comment:
Do we need this, and if so what value should we use?
800px seems like it could be both too wide (e.g. for solid text) and too narrow (e.g. for some content that is better shown full screen, e.g. an image, or interactive) at the same time.

<img width="1228" alt="screen shot 2018-04-20 at 12 48 19 am" src="https://user-images.githubusercontent.com/163966/39031322-7110ca32-4435-11e8-9c9b-3a4b740762b2.png">

I guess chef authors can always adjust their chef-wide template, or override on a case by base basis.